wrapper/src/main/java/software/amazon/jdbc/ConnectionPluginChainBuilder.java

```diff
@@ -62,6 +62,7 @@ public class ConnectionPluginChainBuilder {
           put("logQuery", LogQueryConnectionPluginFactory.class);
           put("dataCache", DataCacheConnectionPluginFactory.class);
           put("efm", HostMonitoringConnectionPluginFactory.class);
+          put("efm2", software.amazon.jdbc.plugin.efm2.HostMonitoringConnectionPluginFactory.class);
```

Comment:
oooh, that looks a bit ugly. Users that would perhaps write custom code might get both plugins classes mixed up.

What about having a complete new name for the efm2 classes?

Reply:
The idea is that efm2 to completely replace efm plugin. I hope such confusions will go away soon.